metabase/metabase
# fix trend lines scaling

Closes https://github.com/metabase/metabase/issues/47351

### Description

Introduced in https://github.com/metabase/metabase/pull/43668. The scaling formatting setting implementation has changed significantly in the PR and updating trend lines to support it got missing.

This PR scales the trend lines dataset to match the original scaled dataset when scaling settings are applied.

### How to verify

- New -> Orders: Count group by Created At -> make it a line chart
- Enable trend lines
- Go to the "Data" tab, under "Y-axis" section open the formatting settings for "Count"
- Set "Multiply by a number" to 0.01 
- Ensure the trend line appears correctly

### Demo

Before
<img width="1120" alt="Screenshot 2024-08-28 at 1 38 17 PM" src="https://github.com/user-attachments/assets/a35b2305-c4f6-402f-9be7-2a41807f91ba">

After
<img width="1488" alt="Screenshot 2024-08-28 at 1 38 01 PM" src="https://github.com/user-attachments/assets/9a3321dd-3520-4db1-b303-4c3297b8d18b">


### Checklist
